### ISSUE TITLE
Stream to the right, first point on the left

### ISSUE THREAD

**Issue by snahedis** [nagix/chartjs-plugin-streaming#112]
Hi !
I want to animate the graph to the right but without reversing the axis. The first point would be on the left. 
Is it currently possible ? And if it's not could it be easily achieve if I fork the plugin or should I rather create another plugin directly ?

Thanks !

**Comment by gotibhai** [comment]
+1 Curious if this is possible

**Comment by nagix** [comment]
I'm not sure how the graph would be animated to the right without reversing the axis. Isn't [this sample](https://nagix.github.io/chartjs-plugin-streaming/samples/reverse.html) what you expect?

**Comment by GeoBrodas** [comment]
Yeah want his badly as well, my solution @nagix to your question, 

I guess we can keep the axis stationary, ie: display the time for let's say 1 minute, and when the line chart reaches the extreme right. that is when the x-axis starts moving to the left while the line chart progresses.

Hope you could picture the process.

**Comment by nikhileshcr** [comment]
Hi, 
Did anyone manage to achieve this? Would appreciate the assistance.

Thanks.
